### PR TITLE
axes updates feature

### PR DESCRIPTION
**Pull request by faizan-khan-iit**
Add the axes updates feature paragraph.

### CODE REVIEW

File: animint.Rmd [hunk 1]
```diff
@@ -365,6 +365,8 @@ The compiler also uses ggplot2 to convert data variables (e.g. life expectancy a
...
 
+animint also supports axes updates for both x and y axes for selected geoms. The idea is that when the user selects a subset of data to be displayed in a plot, the axes are re-calculated so that the displayed data covers the whole of the viewing area in the plot. TODO: add an appropriate example highlighting the use of axes updates (maybe one of Toby's genomic data examples?). The axes updates currently work only for single selection variables, as the axes ranges are computed in the compiler and stored in the JSON plot meta-data file .The renderer uses these stored ranges to update the axes scales everytime the displayed subset of data changes. Calculating all the subset combinations in the compiler for multiple selection variables is not efficient, in which case it would be beneficial to shift this computation to the renderer itself.
+
```
Comment (cpsievert) [inline animint.Rmd]: @faizan-khan-iit do you have any intention on adding this example? I'm hoping to submit soon

Comment (tdhock) [inline animint.Rmd]: maybe this example http://bl.ocks.org/faizan-khan-iit/raw/0c651d949d8d1951a8f1d1de5901abc1/ source code: https://github.com/tdhock/animint/blob/master/inst/examples/Tornadoes_works.R#L61 or look in tests-renderer4-update-axes*.R

Comment (faizan-khan-iit) [inline animint.Rmd]: I think the tornadoes viz is a good example. We should use that.

In the paper, do we want 
1. both the pictures+code of the example, or
2. a gist with the code and link to the animation?

I think the animation will get the idea across better. Any thoughts?

Comment (cpsievert) [inline animint.Rmd]: No need to put the code in the paper, but please to link to code/gist/video. You could also include a couple screengrabs in the paper just to show how the axes have chaged. I have a vimeo account if you need help hosting a video.

Comment (caijun) [inline animint.Rmd]: @faizan-khan-iit the tornadoes example is already included in the ms as Figure 6. I think you can update Figure 6 by adding the axis updates feature you contributed.

**Comment by faizan-khan-iit** [comment]
@cpsievert Ah, It looks like I started off after @caijun had made his commit. So the PR to #10 would be messy. Let me recommit into a new branch in my fork.

**Comment by faizan-khan-iit** [comment]
See #11

**Pull request closed without merging**